modules/core/src/main/kotlin/datamaintain/core/script/TagMatcher.kt

```diff
@@ -38,10 +38,21 @@ class TagMatcher(val tag: Tag, val globPaths: Iterable<String>) {
     companion object {
         @JvmStatic
         fun parse(name: String, pathMatchers: String): TagMatcher {
-            return TagMatcher(Tag(name = name),
-                    pathMatchers.split(",")
-                            .map { pathMatcher -> pathMatcher.trim('[', ']', ' ') }
-                            .toSet())
+            return TagMatcher(Tag(name = name), pathMatchers.split(",")
+                    .map { pathMatcher -> pathMatcher.trim('[', ']', ' ') }
+                    .onEach { if (isNotAbsolute(it)) { throw  NotAbsolutePathMatcherPathException(name, it)} }
+                    .toSet())
+        }
+
+        @JvmStatic
+        private fun isNotAbsolute(pathMatcher: String): Boolean {
+            return pathMatcher.startsWith("~") || pathMatcher.contains("../")
```

Comment:
Case ./ is missing. I don't know if other cases exists but with these 3 cases I guess it is enough.